Sources/snowman/main.swift

```diff
@@ -2,11 +2,17 @@ import SnowManLogic
 import SnowManDictionary
 
 let dictionaryManager = DictionaryManager(words: SnowManDictionary.words) 
-let guessManager = GuessManager(totalIncorrectGuessesAllowed: 10)
 let word = dictionaryManager.getRandomWord()
 
 let io = IO()
+
 let view = View(io: io)
+
+let maxNumberOfGuesses = 10
+view.promptNumberOfGuesses()
+let numberOfGuesses = view.receiveNumberOfGuesses(maxNumberOfGuesses)
```

Comment:
Hmm.. Ok so I think that the number of guesses is an attribute of the `Game`. A game has a number of guesses. But I removed number of guesses from the `Game` class because it was supposed to be established before the game is created. Now should I not establish the number of guesses _before_ the game is created and establish that as part of the creating of the game a la:
```
let game = Game(word: word, guessManager: guessManager, view: view, numberOfGuesses: Int(view.receiveNumberOfGuesses()))
```